hack/check_fixed_issues_references.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
I thought it would fit better into `scripts` directory as we have more CI-related stuff there, but have no hard feelings about this being placed under `hack` as it's not clear now what these directories are really for. 

I suggest we solve that in a separate issue: https://github.com/Kong/kubernetes-ingress-controller/issues/4829

Reply:
I've fixed the error, PTAL